### PR TITLE
Create cryptoScraper.py

### PR DESCRIPTION
**Pull request by Atharvat**
Hi, I have made a function to get the price for different cryptocurrencies from the nomics api (this seems the best one in the free ones). The function currently returns the price for the coin in USTD. Also, please inform if any data other than the price is required to be added.
Thank You

### CODE REVIEW

File: CryptoPy/cryptoScraper.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        data = response.json()
+        for dp in data:
+            price = dp["price"]
+            # print(dp["symbol"] + ': ' + price)
+        return price
+
+    except (ConnectionError, Timeout, TooManyRedirects) as e:
+        print(e)
+
+newPrice = getPrice(key = '2a106acc5e2e1abb424ae177ba6bf929', currencySymbol= 'BTC')
```
Comment (priyanshux) [inline CryptoPy/cryptoScraper.py]: You should never post API keys on public GitHub repositories. Replace it with 'ENTER YOUR API KEY'

File: CryptoPy/cryptoScraper.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    try:
+        response = requests.get(url, params=parameters)
+        # print(response.json())
+        data = response.json()
+        for dp in data:
+            price = dp["price"]
+            # print(dp["symbol"] + ': ' + price)
+        return price
+
```
Comment (priyanshux) [inline CryptoPy/cryptoScraper.py:28]: Change the for loop to this -
`price = [dp["price"] for dp in data]`
        `return(', '.join(price))`

File: CryptoPy/cryptoScraper.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import requests
+import json
```
Comment (priyanshux) [inline CryptoPy/cryptoScraper.py]: remove `import json`

File: CryptoPy/cryptoScraper.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+from requests.exceptions import ConnectionError, Timeout, TooManyRedirects
+
+
+url = 'https://api.nomics.com/v1/currencies/ticker'
+
+
+
```
Comment (priyanshux) [inline CryptoPy/cryptoScraper.py:12]: Remove unnecessary blank spaces

File: CryptoPy/cryptoScraper.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import requests
+import json
+from requests.exceptions import ConnectionError, Timeout, TooManyRedirects
+
```
Comment (priyanshux) [inline CryptoPy/cryptoScraper.py:5]: Create a variable `Currency = 'BTC'` at line 4 and replace `'BTC'` with `Currency`at line 32

**Review by Atharvat — COMMENTED** [review]
I have made the changes as you requested, please review it

**Pull request merged**